bin/www

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env node
+const path = require('path');
+const rootPath = path.join(__dirname, '../');
+
+/* App startup script. */
+
+/* Enables Babel for the server-side code (with exception of this very file). */
+require('@babel/register')({
+  root: rootPath,
+  ignore: [
+    /node_modules\/(?!appirio-tech.*|topcoder|tc-)/,
+    /node_modules\/topcoder-react-utils/,
+  ],
+  "presets": [
+      "@babel/preset-react",
+      [
+        "@babel/preset-env",
+        {
+          "targets": {
+            "esmodules": true
+          }
+        }
+      ]
+    ]
```

Comment:
@vikasrohit just would like to heads up regarding the babel config for the server. It's doesn't match the one which is used by the client-side code, see https://github.com/topcoder-platform/challenge-engine-ui/blob/develop/config/webpack.config.js#L326-L382. As we process the same files on the client and server, we might use some feature in the code which would work using one babel config but fail for another one. Though, as most likely we would use just very simple files on the server, it would hardly happen in real life.

Also, I think if it ever happens, we would know it immediately and code would just fail to compile, so we should be safe from "silent" issues here.

Reply:
Makes sense. Thanks for the heads up. For now, the server side code is just trying to make couple of API requests which are plan javascript functions with basic usage of async-await and few es6 features. None of the React code is actually parse at server side, so I think we might be save from that point.
I might need your help in syncing the babel config to both sides.